packages/jumpstarter-driver-flashers/oci_bundles/buildroot/replace_kernel.sh

```
@@ -0,0 +1,69 @@
+
+url="https://cbs.centos.org/kojifiles/packages/kernel-automotive"
+url="$url/$(echo $KVER | sed -r 's|-|/|; s|\.([^.]*)$|/\1|')"
+pkgs=(
+	kernel-automotive-core-$KVER.rpm
+	kernel-automotive-modules-$KVER.rpm
+	kernel-automotive-modules-core-$KVER.rpm
+)
+
+
+# fetch kernel rpm packages
+for pkg in ${pkgs[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)